android/build.gradle

```diff
@@ -38,32 +31,15 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties["RNTSketchCanvas_" + name]).toInteger()
 }
 
-def supportsNamespace() {
```

Comment:
This change looks unrelated to new arch / old arch. Its just driven by the version of Android Gradle.